### ISSUE TITLE
Cannot turn off autoindent at cli in Ipython 6

### ISSUE THREAD

**Issue by williamjameshandley** [ipython/ipython#11257]
running 
```bash
ipython --no-autoindent
````
does not turn off autoindentation.

neither does running
```ipython
In[1] %autoindent%
autoindent is deprecated since IPython 5: you can now paste multiple lines without turning autoindentation off.
Automatic indentation is: OFF
```
in the ipython interpreter.

I would like to be able to turn off autoindentation for the purposes of sending text to ipython in a sendkeys-like interface for [this project](https://github.com/williamjameshandley/vimteractive). Currently the only way I have of doing this is to start ipython in 
```bash
ipython --simple-prompt
```
but then I lose functionality like coloring/autocompletion.

**Comment by Carreau** [comment]
Does something like https://github.com/ipython/rlipython fit your needs ? 
My guess is you actually want to bypass all the  IPython Gui and just send code to be executed.

**Comment by williamjameshandley** [comment]
Unfortunately this doesn't seem to fix my issue, as when I try start ipython with
`ipython --TerminalIPythonApp.interactive_shell_class=rlipython.TerminalInteractiveShell`, and then try to send either individual lines, or multiple lines at once via vim's `term_send_keys`, ipython still tries to insert additional tabs

**Comment by Carreau** [comment]
Does #11273 fit your needs ?

**Comment by williamjameshandley** [comment]
Yes! That seems to fix all my use cases just by passing `--no-autoindent`

**Comment by Carreau** [comment]
Closed by #11273

**Comment by eromoe** [comment]
How do I config this in ipython config file ??
